synfig-core/src/synfig/main.cpp

```diff
@@ -202,7 +202,7 @@ synfig::Main::Main(const synfig::String& basepath,ProgressCallback *cb):
 
 	// Paths
 
-	root_path       = etl::dirname(basepath);
```

Comment:
In this case, you also need to fix `synfig-core/src/tool/main.cpp`.
As for now it can't find files:
```
debug/output/Debug/bin/synfig: warning: Unable to find module "lyr_freetype" (file not found)
Unable to load module 'lyr_freetype'
```

Reply:
"Whoops I did it again" :(